src/index.js

```diff
@@ -9,13 +9,20 @@ const IS_WINDOWS = platform() === 'win32';
 
 // Helper functions
 const noop = () => null;
-const matches = (key, importee) => {
+const matches = (key, importee, isRegEx) => {
```

Comment:
Maybe `key` should be renamed as it no longer is an object key? E.g. `pattern`?

Reply:
Agreed. Done.